src/client/entity-editor/name-section/name-section.js

```diff
@@ -187,7 +187,7 @@ class NameSection extends React.Component {
 											(
 												<ListGroupItem
 													bsStyle="warning"
-													href={`/${entityType}/${match.bbid}`}
+													href={`/${_kebabCase(entityType)}/${match.bbid}`}
```

Comment:
As you can see below there is an error reported: `'_kebabCase' is not defined`.

In the other file we imported it with the name `_kebabCase`, but in this one we import the whole lodash library as `_` (https://github.com/bookbrainz/bookbrainz-site/blob/master/src/client/entity-editor/name-section/name-section.js#L44).

So you can call the method like this: `_.kebabCase` with the extra dot.

Reply:
Thanks @MonkeyDo for the feedback. I'll try not to make careless mistakes like these in the future.